### ISSUE TITLE
Add examples to docstrings for consumer methods

### ISSUE THREAD

**Issue by rnowling-memphis** [memphisdev/memphis.py#181]
The method `Consumer.fetch()` has an example of how to use the method in its [docstring](https://github.com/memphisdev/memphis.py/blob/master/memphis/consumer.py#L118).  Let's add a similar example to the docstring for [`Consumer.consume()`](https://github.com/memphisdev/memphis.py/blob/master/memphis/consumer.py#L55).

**Comment by rnowling-memphis** [comment]
@Adarsh-jaiss are you interested in this?

**Comment by Adarsh-jaiss** [comment]
- yeah I would love to work on this! 
- could you please explain me a bit about the issue?

**Comment by rnowling-memphis** [comment]
Please compare the docstrings (function documentation) for the two functions I put links to.  One of them has a nice example of how to call the function, while the other does not :)

**Comment by Adarsh-jaiss** [comment]
Will it work, kindly review this : 

```
def consume(self, callback):
    """
    Consume events.

    This method starts consuming events from the specified station and invokes the provided callback function for each batch of messages received.

    Parameters:
        callback (function): A function that will be called with each batch of messages received. The function should have the following signature:
            - `callback(messages: List[Message], error: Optional[MemphisError], context: Dict) -> Awaitable[None]`
            - `messages`: A list of `Message` objects representing the batch of messages received.
            - `error`: An optional `MemphisError` object if there was an error while consuming the messages.
            - `context`: A dictionary representing the context that was set using the `set_context()` method.

    Example:
        import asyncio
        from memphis import Memphis

        async def message_handler(messages, error, context):
            if error:
                print(f"Error occurred: {error}")
                return

            for message in messages:
                print(f"Received message: {message}")

        async def main():
            memphis = Memphis()
            await memphis.connect(host='localhost', username='user', password='pass')
            consumer = await memphis.consumer(station_name='my_station', consumer_name='my_consumer', consumer_group='my_group')
            consumer.set_context({'key': 'value'})
            consumer.consume(callback=message_handler)

            # Keep the event loop running
            while True:
                await asyncio.sleep(1)

        asyncio.run(main())
    """

```
    self.dls_callback_func = callback
    self.t_consume = asyncio.create_task(self.__consume(callback))

**Comment by rnowling-memphis** [comment]
@Adarsh-jaiss It's nearly impossible to review and comment on an issue -- can you create a PR?  I think going forward, it would be best just to go directly to creating a PR if you want feedback on code.  Thanks!

**Comment by rnowling-memphis** [comment]
Closed by #182
